pyramid/scripts/pcreate.py

```diff
@@ -154,5 +169,48 @@ def out(self, msg): # pragma: no cover
         if not self.quiet:
             print(msg)
 
+    def validate_input(self):
+        if not self.options.scaffold_name:
+            self.out('You must provide at least one scaffold name: -s <scaffold name>')
+            self.out('')
+            self.show_scaffolds()
+            return False
+        if not self.args:
+            self.out('You must provide a project name')
+            return False
+        available = [x.name for x in self.scaffolds]
+        diff = set(self.options.scaffold_name).difference(available)
+        if diff:
+            self.out('Unavailable scaffolds: %s' % ", ".join(list(diff)))
+            return False
+
+        pkg_name = self.project_vars['package']
+
+        if pkg_name == 'site' and not self.options.force_bad_name:
+            self.out('The package name "site" has a special meaning in '
+                     'Python. Are you sure you want to use it as your '
+                     'project\'s name?')
+            return self.confirm_bad_name('Really use "{0}"?: '.format(pkg_name))
+
+        # check if pkg_name can be imported (i.e. already exists in current
+        # $PYTHON_PATH, if so - let the user confirm
+        pkg_exists = True
+        try:
+            __import__(pkg_name, globals(), locals(), [], 0) # use absolute imports
+        except ImportError as error:
+            pkg_exists = False
+        if not pkg_exists:
+            return True
+
+        if self.options.force_bad_name:
+            return True
+        self.out('Package "{0}" already exists, are you sure you want '
```

Comment:
I think "A package named "{0}" already exists..." is better than saying "Package foo already exists" as the latter implies that it's the package you want, not some other package.